### DISCUSSION TITLE
Nuxt 3 Support?

### DISCUSSION THREAD

**Discussion by johnkomarnicki** [nuxt/content#1037] [General]
Any time frame on a release for support with Nuxt 3?

**Comment by snakeshift** [comment]
I want information about nuxt/content v2.

**Comment by benoitdemaegdt** [comment]
Hey there 👋 

You can have a look to [this issue](https://github.com/nuxt/content/issues/974).

And according to [this tweet](https://twitter.com/Atinux/status/1523245439054606336) from Atinux, the release for Nuxt3 support should land very soon.

**Comment by lucpotage** [comment]
In case you missed it, version 2 was released 2 days ago: https://content.nuxtjs.org/blog/announcing-v2
